msc_pygeoapi/loader/nwp_dataset_footprints.py

```diff
@@ -36,7 +36,7 @@
 import logging
```

Comment:
Imports should be in the order of standard library, third party packages, and local imports, alphabetically, grouped by spaces.